### DISCUSSION TITLE
Is Prism supported in Uno ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [unoplatform/uno#8593] [Q&A]
Is there a way to continue to use the Prism library with WinUI and hence Uno ? [This](https://github.com/PrismLibrary/Prism/discussions/2523) is currently a blocking issue.

**Comment by jeromelaban** [comment]
There is nothing we can do at this point. The support has been merged in Prism, but there's yet to be a new release of Prism which supports Uno 4.x. When this gets released you'll be able to use it. Until then, you'll need to build it yourself.
